docs/1.getting-started/5.routing.md

```diff
@@ -38,6 +38,16 @@ pages/
 
 ::
 
+To display pages, use the `<NuxtPage />` tag:
+
+```vue [app.vue]
+<template>
+  <div>
+    <NuxtPage />
+  </div>
+</template>
+```
```

Comment:
A lot more documentation about this is currently in https://nuxt.com/docs/guide/directory-structure/pages. Maybe we could add a link with a line about `app.vue` rather than put this example here?

Reply:
Ah, didn't find that page. Excellent 👌  I updated. Hope it's correct :)